docs/introduction.md

```diff
@@ -52,8 +52,8 @@ have their own set of options.
 
 ### SAT Container Environment
 
-The `sat` command-line utility runs in a container using podman, a daemonless container runtime. SAT runs on Kubernetes
-manager nodes. A few important points about the SAT container environment include the following:
+The `sat` command-line utility runs in a container using Podman, a daemon-less container runtime. SAT runs on
```

Comment:
given that Podman's website uses "daemonless", this is also one I'd consider adding and exception for.